src/utils/validators.ts

```diff
@@ -60,3 +51,26 @@ export const adjoiningAreaValidator = helpers.withMessage('В назві маю
 
 const validMainHouseInfo = helpers.regex(/^\d+$/);
 export const mainHouseInfoValidator = helpers.withMessage('В назві мають бути цифри від 0 до 9', validMainHouseInfo);
+
+export const addressValidator = helpers.withMessage(`${ukrLangMessage}`, ukranianLan);
+
+export const houseNumAndHouseBlockValidator = helpers.withMessage(
+  `${ukrLangMessage} або цифри від 0 до 9`,
+  validHouseNumAndHouseBlock
+);
+const validZipCode = helpers.regex(/^\d{5}$/);
+export const zipCodeValidator = helpers.withMessage('В назві мають бути 5 цифр від 0 до 9', validZipCode);
+
+const validPhoneNumber = helpers.regex(/^\+380\d{3}\d{2}\d{2}\d{2}$/);
```

Comment:
Why do you changed validPhoneNumber validator? Won't it break existing logic in app?

Reply:
fixed